website/source/docs/internals/architecture.html.md

```diff
@@ -61,11 +61,12 @@ clarify what is being discussed:
   availability. Servers federate across regions to make Nomad globally aware.
 
 * **Regions and Datacenters** - Nomad models infrastructure as regions and datacenters.
-  Regions may contain multiple datacenters. Servers are assigned to regions and manage
-  all state for the region and make scheduling decisions within that region. Requests that
-  are made between regions are forwarded to the appropriate servers. As an example, you may
-  have a `US` region with the `us-east-1` and `us-west-1` datacenters, connected to the
-  `EU` region with the `eu-fr-1` and `eu-uk-1` datacenters.
+  Regions may contain multiple datacenters. For example, you may have a `US` region with the 
+  `us-east-1` and `us-west-1` datacenters, connected to the `EU` region with the `eu-fr-1` and 
```

Comment:
I like the old copy more. It mentions federation before saying two regions are "connected"